samples/stock/demo/all-indicators/demo.js

```diff
@@ -0,0 +1,193 @@
+        ohlc.push([
+            data[i][0], // the date
+            data[i][1], // open
+            data[i][2], // high
+            data[i][3], // low
+            data[i][4] // close
+        ]);
+
+        volume.push([
+            data[i][0], // the date
+            data[i][5] // the volume
+        ]);
+    }
+
+    // create the chart
+    const chart = Highcharts.stockChart('container', {
+        chart: {
+            height: 600
+        },
+        title: {
+            text: 'AAPL Historical'
+        },
+        subTitle: {
+            text: 'All indicators'
+        },
+        legend: {
+            enabled: true
+        },
+        rangeSelector: {
+            selected: 2
+        },
+        tooltip: {
+            split: true
+        },
+        yAxis: [{
+            height: '70%'
+        }, {
+            top: '75%',
+            height: '25%'
+        }],
+        series: [{
+            type: 'candlestick',
+            id: 'aapl',
+            name: 'AAPL',
+            data: data
+        }, {
+            type: 'column',
+            id: 'volume',
+            name: 'Volume',
+            data: volume,
+            yAxis: 1
+        }]
+    });
+
+    document.getElementById("overlays").addEventListener("change", function (e) {
+        const seriesToAdd = e.target.value;
+
+        // Destroy previously existing series.
+        if (chart.series[2] && chart.series[2].yAxis.options.index === 0) {
+            chart.series[2].destroy(false);
+        }
+        if (chart.series[3] && chart.series[3].yAxis.options.index === 0) {
+            chart.series[3].destroy(false);
+        }
+
+        if (seriesToAdd !== 'none') {
+            if (seriesToAdd === 'ikh') {
+                chart.addSeries({
+                    type: 'ikh',
+                    linkedTo: 'aapl',
+                    showInLegend: true,
+                    tenkanLine: {
+                        styles: {
+                            lineColor: 'lightblue'
+                        }
+                    },
+                    kijunLine: {
+                        styles: {
+                            lineColor: 'darkred'
+                        }
+                    },
+                    chikouLine: {
+                        styles: {
+                            lineColor: 'lightgreen'
+                        }
+                    },
+                    senkouSpanA: {
+                        styles: {
+                            lineColor: 'green'
+                        }
+                    },
+                    senkouSpanB: {
+                        styles: {
+                            lineColor: 'red'
+                        }
+                    },
+                    senkouSpan: {
+                        color: 'rgba(0, 255, 0, 0.3)',
+                        styles: {
+                            fill: 'rgba(0, 0, 255, 0.1)'
+                        }
+                    }
```

Comment:
Do we need to define all styles for lines in IKH?

Reply:
No, we don't have to, actually most of them (especially the fill) are set by default.